osgp/platform/osgp-adapter-kafka-distributionautomation/src/test/java/org/opensmartgridplatform/adapter/kafka/da/application/mapping/ScadaMeasurementPublishedEventConverterTest.java

```diff
@@ -105,4 +110,34 @@ void testNullString() {
         assertThat(event).isNull();
     }
 
+    @Test
+    void testEsm() {
```

Comment:
Were this test and the helper method below it added accidentally with the other changes?
If not, it might be good to make it a little clearer and actually assert something instead of printing the length to system out.